src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -463,17 +463,17 @@ const UPDATE_RESULT = new GraphQLObjectType({
   fields: UPDATE_RESULT_FIELDS,
 });
 
-const CLASS_FIELDS = {
+const CLASS_OBJECT_FIELDS = {
   ...CREATE_RESULT_FIELDS,
   ...UPDATE_RESULT_FIELDS,
   ...INPUT_FIELDS,
 };
 
-const CLASS = new GraphQLInterfaceType({
-  name: 'Class',
+const CLASS_OBJECT = new GraphQLInterfaceType({
```

Comment:
Would "ParseObject" be suitable? I believe it's more descriptive than ClassObject.